contracts/core/EntryPoint.sol

```diff
@@ -141,6 +149,24 @@ contract EntryPoint is IEntryPoint, StakeManager {
         _compensate(beneficiary, collected);
     }
 
+    function simulateExecution(UserOperation calldata op) external override {
+
+        UserOpInfo memory opInfo;
+
+        (uint256 deadline, uint256 paymasterDeadline,) = _validatePrepayment(0, op, opInfo, SIMULATE_FIND_AGGREGATOR);
+        //ignore signature check failure
+        if (deadline == SIG_VALIDATION_FAILED) {
+            deadline = 0;
+        }
+        if (paymasterDeadline == SIG_VALIDATION_FAILED) {
+            paymasterDeadline = 0;
+        }
+        _validateDeadline(0, opInfo, deadline, paymasterDeadline);
+        numberMarker();
```

Comment:
It is hard to keep track of where these `numberMarker`s belong and why. Why is this one here? Maybe it should accept some string so it looks like `numberMarker("paymaster context");` or something.

Reply:
factory
<marker>
validateUserOp
<marker>
validatePaymasterUserOp
<marker>
exec 
maybe adding comments would help (as helpful as a string, since we're not going to do anything with it anyway)